app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,111 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.home.HomeActivity;
+import ar.com.wolox.android.example.ui.signup.SignUpActivity;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
+
+import static ar.com.wolox.android.example.utils.Constants.WOLOX_WEB;
+
+/**
+ * Login fragment
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @BindView(R.id.vUserEmailInput)
+    EditText vUserEmailInput;
+
+    @BindView(R.id.vUserPasswordInput)
+    EditText vUserPasswordInput;
+
+    @BindView(R.id.vLoginButton)
+    Button vLoginButton;
+
+    @BindView(R.id.vSignUpButton)
+    Button vSignUpButton;
+
+    @BindView(R.id.vLoginTermsAndConditions)
+    TextView vLoginTermsAndConditions;
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        loadLocalLoginData();
+    }
+
+    @Override
+    public void setListeners() {
+        vLoginButton.setOnClickListener(v -> attemptToLogin());
+        vLoginTermsAndConditions.setOnClickListener(v -> openTermsAndConditions());
+        vSignUpButton.setOnClickListener(v -> openSignUpPage());
+    }
+
+    private void attemptToLogin() {
+        String userEmail = vUserEmailInput.getText().toString();
+        String password = vUserPasswordInput.getText().toString();
+        getPresenter().validateForm(userEmail, password);
+    }
+
+    private void openTermsAndConditions() {
+        Intent i = new Intent(Intent.ACTION_VIEW);
+        i.setData(Uri.parse(WOLOX_WEB));
```

Comment:
You can use HTML on the Terms and condition string resource and then set to the view this: 
**setMovementMethod(LinkMovementMethod.getInstance());**

Reply:
Great, I changed it, and removed the method openTermsAndConditions because is not longer necessary.